freqtrade/rpc/api_server/ws/proxy.py

```diff
@@ -27,20 +27,23 @@ async def send(self, data):
         """
         Send data on the wrapped websocket
         """
-        if isinstance(data, str):
-            data = data.encode()
 
-        if hasattr(self._websocket, "send_bytes"):
-            await self._websocket.send_bytes(data)
+        if not isinstance(data, str):
+            # We use HybridJSONWebSocketSerializer, which when serialized returns
+            # bytes because of ORJSON, so we explicitly decode into a string
+            data = str(data, "utf-8")
```

Comment:
i think this should be moved to the serializer (we expect a string out of that).

otherwise we might run into unexpected problems should we use a different serializer for whatever reason in the future (maybe by allowing the client to switch to a different format).